crypto/x509/x509_test.cc

```diff
@@ -2918,8 +2919,33 @@ TEST(X509Test, Dilithium3SignVerifyCert) {
   EVP_DigestSignInit(md_ctx.get(), nullptr, nullptr, nullptr, pkey);
   ASSERT_TRUE(X509_sign_ctx(leaf.get(), md_ctx.get()));
 
+  bssl::UniquePtr<BIO> bio(BIO_new(BIO_s_mem()));
+
+  //FILE *fptr;
```

Comment:
Looks like you've got some commented code here to clean up before merging.

Reply:
Thank you, good catch. I was using this test to output my test Dilithium certificates. Removed now.